packages/firebase_messaging/firebase_messaging/example/test_driver/instance_e2e.dart

```diff
@@ -0,0 +1,146 @@
+// Copyright 2020, the Chromium project messagingors.  Please see the MESSAGINGORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io' show Platform;
+import 'package:firebase_messaging/firebase_messaging.dart';
+import 'package:firebase_core/firebase_core.dart';
+import 'package:flutter_test/flutter_test.dart';
+import './test_utils.dart';
+
+bool SKIP_MANUAL_TESTS = false;
```

Comment:
should be set to true to skip the tests that should run manually. Not sure if there's a way to have this as an env var. So remotely in github actions, it's always true. 

Using this flag is an alternative to having the manual tests commented out

Reply:
Resolved on latest commit using dart defines, thanks :)

Reply:
Nice!! I was looking into that for flutter_auth, wasn't sure if they actually worked.

Reply:
Aye seems to work nicely :) here's the specific commit for snooping purposes: [`dfc116c` (#4012)](https://github.com/FirebaseExtended/flutterfire/pull/4012/commits/dfc116c8c2475a45906eecdcdd89da8b597ab3de) 🤓